packages/ui-date-time-input/src/DateTimeInput/README.md

```diff
@@ -63,9 +63,14 @@ class Example extends React.Component {
   }
 
   onChange = (e, isoDate) => {
+    let messages = []
```

Comment:
Now the example is in "invalid mode" when the page loads. Is that a good thing? I would assume it is a valid pattern that on load the input can be empty before the user touches it, it is weird that it already loads with error. Imho...

Reply:
hmm the error message is definitely wrong in this case, I'll fix that. Also you are right, usually required form fields only start to complain about not having a value after you moved the focus out of them, I'll try to fix it.